libraries/redcore/api/soap/soap.php

```diff
@@ -148,16 +156,13 @@ public function execute()
 	 */
 	public function apiSoap()
 	{
+		$wsdl = JUri::root() . $this->wsdlPath;
 		$params = array(
+			'uri' => $wsdl,
```

Comment:
yeah it's optional, but I'm defining it everywhere in the wsdl for namespace, etc